math/binomialcoefficient_test.go

```diff
@@ -0,0 +1,39 @@
+// binomialcoefficient_test.go
+// description: Returns C(n, k)
+// author: Akshay Dubey (https://github.com/itsAkshayDubey)
+// see binomialcoefficient.go
+
+package math_test
+
+import (
+	"testing"
+
+	algmath "github.com/TheAlgorithms/Go/math"
+)
+
+func TestC(t *testing.T) {
+	var tests = []struct {
+		n        int
+		k        int
+		expected int
+	}{
+		{5, 2, 10},
+		{7, 4, 35},
+		{0, 0, 1},
+		{-1, 1, -1},
+		{1, -1, -1},
+		{-1, -1, -1},
```

Comment:
These are all errors but you're not checking whether you actually even get errors. If you define a new error variable 

```Go
var ErrPosArgsOnly error = errors.New("arguments must be positive")
```
and change the return statement in line 20 of the implementation file to 

```
    return -1, ErrPosArgsOnly
```
then you can check that indeed you get that exact error in this table test.